server/src/main/java/org/apache/iotdb/db/engine/compaction/utils/log/CompactionLogger.java

```diff
@@ -30,47 +29,41 @@
 import java.util.List;
 
 /** MergeLogger records the progress of a merge in file "merge.log" as text lines. */
-public class RewriteCrossSpaceCompactionLogger implements AutoCloseable {
+public class CompactionLogger implements AutoCloseable {
 
-  public static final String COMPACTION_LOG_NAME = "cross-compaction.log";
-  public static final String COMPACTION_LOG_NAME_FEOM_OLD = "merge.log";
+  public static final String CROSS_COMPACTION_LOG_NAME = "cross-compaction.log";
+  public static final String CROSS_COMPACTION_LOG_NAME_FEOM_OLD = "merge.log";
```

Comment:
should be `CROSS_COMPACTION_LOG_NAME_FROM_OLD`

Reply:
Resolved.